src/Common/Help_Hub/Hub.php

```diff
@@ -0,0 +1,423 @@
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function generate_iframe_content(): void {
+		$page   = tribe_get_request_var( 'page' );
+		$iframe = tribe_get_request_var( 'embedded_content' );
+
+		// Return early if $page is empty or not 'tec-events-help-hub', or if $iframe is empty.
+		if ( empty( $page ) || 'tec-events-help-hub' !== $page || empty( $iframe ) ) {
+			return;
+		}
+
+		// Enqueue our assets for the Iframe.
+		add_action( 'wp_enqueue_scripts', [ $this, 'enqueue_help_page_iframe_assets' ] );
+
+		// Disable the admin bar for iframe requests.
+		// phpcs:ignore WordPressVIPMinimum.UserExperience.AdminBarRemoval.RemovalDetected
+		show_admin_bar( false );
+		// Render the iframe content.
+		$this->render_template(
+			'help-hub/support/iframe-content'
+		);
+
+		exit;
+	}
+
+	/**
+	 * Enqueues the necessary scripts for the help page and dequeues all theme styles.
+	 *
+	 * @since TBD
+	 * @return void
+	 */
+	public function enqueue_help_page_iframe_assets(): void {
+		define( 'IFRAME_REQUEST', true );
+		tribe_asset(
+			Tribe__Main::instance(),
+			'help-hub-iframe-style',
+			'help-hub-iframe.css',
+			null,
+			'wp_enqueue_scripts'
+		);
+		tribe_asset(
+			Tribe__Main::instance(),
+			'help-hub-iframe-js',
+			'admin/help-hub-iframe.js',
+			null,
+			'wp_enqueue_scripts',
+			[
+				'localize' => [
+					'name' => 'helpHubSettings',
+					'data' => [
+						'docsbot_key'    => $this->config->get( 'DOCSBOT_SUPPORT_KEY' ),
+						'zendeskChatKey' => $this->config->get( 'ZENDESK_CHAT_KEY' ),
+					],
+				],
+			]
+		);
+		$this->dequeue_theme_styles();
+	}
+
+	/**
+	 * Dequeues theme-related styles to avoid conflicts within the iframe.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	protected function dequeue_theme_styles(): void {
+		global $wp_styles;
+		$theme_directory = get_template_directory_uri();
+
+		// Dequeue only styles from the current theme.
+		foreach ( $wp_styles->queue as $handle ) {
+			$src = $wp_styles->registered[ $handle ]->src;
+			if ( strpos( $src, $theme_directory ) !== false ) {
+				wp_dequeue_style( $handle );
+			}
+		}
+	}
+
+	/**
+	 * Generates the admin notice HTML.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The HTML for the admin notice.
+	 */
+	private function generate_notice_html(): string {
+		$notice_slug    = 'tec-common-help-chatbot-notice';
+		$notice_content = sprintf(
+		// translators: 1: the opening tag to the chatbot link, 2: the closing tag.
+			_x(
+				'To find the answer to all your questions use the %1$sTEC Chatbot%2$s',
+				'The callout notice to try the chatbot with a link to the page',
+				'tribe-common'
+			),
+			'<a data-tab-target="tec-help-tab" href="#">',
+			'</a>'
+		);
+
+		$notice_admin = ( new AdminNotice( $notice_slug, "<p>$notice_content</p>" ) )
+			->urgency( 'info' )
+			->inline()
+			->dismissible()
+			->withWrapper();
+
+		return AdminNotices::render( $notice_admin, false );
```

Comment:
Does this feel ready to use here?

Reply:
I believe so. I've gone through the pages a handful of times and the notice library hasn't created any weirdness with the page. The notice renders exactly where we need.
![image](https://github.com/user-attachments/assets/974cbcc5-115c-401b-b392-5ad9de1a8759)